### ISSUE TITLE
[BUG]Not possible to delete connection after upgrade

### ISSUE THREAD

**Issue by aangstrom** [frangoteam/FUXA#1337]
**Describe the bug**
After upgrade to 1.2.0 is not possible to delete 2 connections, visible in the top of attached image with background light gray

**Screenshots**
![image](https://github.com/user-attachments/assets/f2237290-3fb1-4311-8153-07dc78c44316)

**Environment**
 - FUXA version: 1.2.0 1818
 - Node.js version: n/a
 - npm version: n/a
 - Platform/OS: ubuntu/docker
 - Browser: Chrome

**Comment by unocelli** [comment]
Hi Armando, I can't reproduce it, can you please send me the project.
![remove-device](https://github.com/user-attachments/assets/682e4874-3632-4c0a-9790-6506e93c99e9)

**Comment by aangstrom** [comment]
Hi Umberto, 
here attached the project file
[AAngstrom_Fuxa_Project.json](https://github.com/user-attachments/files/16862917/AAngstrom_Fuxa_Project.json)

**Comment by mauroalexandre** [comment]
@unocelli 

The error occurs because there are two devices with the same name. The implementation begins with checking the name to determine which device is being removed.

https://github.com/frangoteam/FUXA/blob/a3b1f2229168ff000ff3ce2968c82a4cd9e2768a/client/src/app/device/device-map/device-map.component.ts#L468

**We have two options:**

**1 -** Do not allow registering devices with the same name (which I believe makes sense)

**2 -** Change the logic of the method

**Comment by unocelli** [comment]
@mauroalexandre I agree with you for option 1, but I thought there was already a validator

**Comment by unocelli** [comment]
@aangstrom try to remove it from the 'Devices list' View (top right button)

**Comment by unocelli** [comment]
maybe the check is not done when importing a device
